feat: Added compatibility for Weather Wonders

seifeldingamal opened the pull request:
I hope this compatibility changes are up to standards.
Here is the [mod link](https://www.nexusmods.com/stardewvalley/mods/23868).

seifeldingamal committed:
feat: Added compatibility for Weather Wonders

I hope this compatibility changes are up to standards.
Here is the mod link: https://www.nexusmods.com/stardewvalley/mods/23868

GStefanowich commented:
From a first glance this looks nice, but I've got two concerns

1. Are the translations machine translated? I see that you've filled in all of them. All the current translations in the mod are from translators here https://github.com/StardewModders/mod-translations/issues/56
    * If so I'd rather leave them out
2. You've added config options for all of the weather options in Weather Wonders, but they're not wrapped if an `if` block to test if the mod is present. I'm fine with supporting weather of different mods, but I'd like to maintain all normalcy for users who don't use those mods

GStefanowich commented:
For point 2 a simple

```csharp
if ( this.Mod.Helper.ModRegistry.IsLoaded("Kana.WeatherWonders") ) {
}
```

should work just fine

seifeldingamal committed:
fix: removed traslations done via machine and added mod conditional for config

seifeldingamal commented:
Thanks for pointing out the concerns.
1- I did de, es, fr myself but am not a translator. I removed all translations except for de as I feel confident enough.
2- I added the suggested conditional to the config file and tested it.

GStefanowich commented:
Thanks, it all looks great otherwise. Nicely done

Going to do some testing with the mod to make sure all is in order, add in a couple of previously requested features, and then I'll package a release
